galite-tests/src/test/kotlin/org/kopi/galite/tests/form/FormWithList.kt

```diff
@@ -234,15 +234,9 @@ class BlockWithManyTables : FormBlock(20, 20, "Test block") {
     columns(m.id, r.module)
   }
 
-  val module = visit(domain = Domain<Int>(20), position = at(1, 1)) {
-    label = "module"
-    help = "module"
-    columns(r.module)
-  }
-
-  val moduleName = visit(domain = Domain<String>(20), position = at(1, 1)) {
```

Comment:
Why you are removing this field?

Reply:
Redundant field, we have mid that contain : columns(m.id, r.module)